features/europe/kosovo/kosovo.geojson

```diff
@@ -0,0 +1,190 @@
+{
+  "type": "Feature",
+  "id": "kosovo",
+  "properties": {},
+  "geometry": {
+    "type": "LineString",
```

Comment:
It is actually supposed to be a "Polygon" but I guess in #193 you had some trouble.. I'll check what might have caused the issue.